pkg/report/app_instance_tags.go

```diff
@@ -0,0 +1,18 @@
+package report
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated-sdk/pkg/store"
+	tags "github.com/replicatedhq/replicated-sdk/pkg/tags"
+	tagstypes "github.com/replicatedhq/replicated-sdk/pkg/tags/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+func SendAppInstanceTags(ctx context.Context, clientset kubernetes.Interface, sdkStore store.Store, tdata tagstypes.InstanceTagData) error {
```

Comment:
what do you think about moving this logic back to the handler and doing the two things there. it might be a bit clearer that way as this `SendAppInstanceTags` function is doing a bit more than just sending tags, it's:
1. saving the instance tags and
2. sending a full instance reporting event